src/index.js

```diff
@@ -1,15 +1,43 @@
 import './css/index.css';
+import Game from './modules/Game.js';
 import Scores from './modules/Scores.js';
 
 const frmAddScore = document.getElementById('frmAddScore');
 const cntScores = document.getElementById('cntScores');
+const btnRefresh = document.getElementById('btnRefresh');
+const cntError = document.getElementById('cntError');
 const scores = new Scores(cntScores);
 
+const game = new Game("jorge's game", cntError);
+game.create();
+
+const btnSubmit = frmAddScore.querySelector('[type="submit"]');
 frmAddScore.addEventListener('submit', (e) => {
   e.preventDefault();
+  btnSubmit.disabled = true;
   const name = frmAddScore.name.value.trim();
   const score = frmAddScore.score.value.trim();
-  scores.add(name, score);
-  frmAddScore.reset();
-  frmAddScore.name.focus();
+  game
+    .addScore(name, Number(score))
+    .then(() => {
+      scores.add(name, score);
+      frmAddScore.reset();
+      frmAddScore.name.focus();
+    })
+    .finally(() => {
+      btnSubmit.disabled = false;
+    });
+});
+
+btnRefresh.addEventListener('click', () => {
+  btnRefresh.disabled = true;
```

Comment:
- [ ] I like how you are disabling your buttons between requests. Great job. :100: